xformers/triton/softmax.py

```diff
@@ -194,16 +192,12 @@ def _softmax_dispatch(
         )
         logging.warning(e)
 
-    if causal and not _triton_registered_warnings:
-        logging.warning(
-            "Triton softmax could not be used. \
-                The causal flags is being passed but it does not provide any benefit with PyTorch softmax."
-        )
-        _triton_registered_warnings = True
-
     if mask is not None:
         x = x + mask
 
+    if causal:
```

Comment:
it feels like this falls into the "silently support something which probably does not provide benefits", no ? You used to have strong feelings about that.